patches/0038-ipq40xx-Add-support-for-CIG-WF610D-AP.patch

```diff
@@ -0,0 +1,691 @@
++					gpios = <&tlmm 48 GPIO_ACTIVE_LOW>;
++					label = "wf6203:green:eth2";
++					default-state = "off";
++			};																
++			wf6203_green_wifi2g {
++					gpios = <&tlmm 40 0>;
++					label = "wf6203:green:wifi2g";
++					default-state = "off";
++			};
++			wf6203_green_wifi5g {
++					gpios = <&tlmm 39 0>;
++					label = "wf6203:green:wifi5g";
++					default-state = "off";
++			};				
++			wf6203_green_status {
++					gpios = <&tlmm 49 0>;
++					label = "wf6203:green:status";
++					default-state = "off";
++			};								
++		};
++
++
++		watchdog@b017000 {
++			timeout-sec = <60>;
++			status = "okay";
++		};
++
++		qpic_bam: dma@7984000 {
++			status = "okay";
++		};
++
++		nand: qpic-nand@79b0000 {
++			pinctrl-0 = <&nand_pins>;
++			pinctrl-names = "default";
++			status = "okay";
++		};
++
++		mdio@90000 {
++			pinctrl-0 = <&mdio_pins>;
++			pinctrl-names = "default";
++			phy-reset-gpio = <&tlmm 18 0>;
++			status = "okay";
++			bias-disable;
++		};
++
++		ess-switch@c000000 {
++			status = "okay";
++//			switch_mac_mode = <0x0>; /* mac mode for RGMII RMII */
++//			switch_initvlas = <
++//				0x0007c 0x54 /* PORT0_STATUS */
++//			>;
++			//led_source@0 {
++			//	led = <0x3>;     /*led number */
++			//	source = <0x1>;  /*source id 1 */
++			//	mode = "normal"; /*on,off,blink,normal */
++			//	speed = "all";   /*10M,100M,1000M,all */
++			//	freq = "auto";   /*2Hz,4Hz,8Hz,auto*/
++			//};
++			//led_source@1 {
++			//	led = <0x4>;     /*led number */
++			//	source = <0x4>;  /*source id 4 */
++			//	mode = "normal"; /*on,off,blink,normal */
++			//	speed = "all";   /*10M,100M,1000M,all */
++			//	freq = "auto";   /*2Hz,4Hz,8Hz,auto*/
++			//};
++			//led_source@2 {
++			//	led = <0x5>;     /*led number */
++			//	source = <0x7>;  /*source id 7 */
++			//	mode = "normal"; /*on,off,blink,normal */
++			//	speed = "all";   /*10M,100M,1000M,all */
++			//	freq = "auto";   /*2Hz,4Hz,8Hz,auto*/
++			//};
++			//led_source@3 {
++			//	led = <0x6>;     /*led number */
++			//	source = <0xa>;  /*source id 10 */
++			//	mode = "normal"; /*on,off,blink,normal */
++			//	speed = "all";   /*10M,100M,1000M,all */
++			//	freq = "auto";   /*2Hz,4Hz,8Hz,auto*/
++			//};
++			//led_source@4 {
++			//	led = <0x7>;     /*led number */
++			//	source = <0xd>;  /*source id 13 */
++			//	mode = "normal"; /*on,off,blink,normal */
++			//	speed = "all";   /*10M,100M,1000M,all */
++			//	freq = "auto";   /*2Hz,4Hz,8Hz,auto*/
++			//};
++		};
++
++		ess-psgmii@98000 {
++			status = "okay";
++		};
++
++		edma@c080000 {
++			status = "okay";
++		};
++
++		usb3_ss_phy: ssphy@9a000 {
++			status = "okay";
++		};
++
++		usb3_hs_phy: hsphy@a6000 {
++			status = "okay";
++		};
++
++		usb3: usb3@8af8800 {
++			status = "okay";
++		};
++
++		usb2_hs_phy: hsphy@a8000 {
++			status = "okay";
++		};
++
++		usb2: usb2@60f8800 {
++			status = "okay";
++		};
++
++		ext_wtd {
++			compatible = "linux,wdt-gpio";
++			gpios = <&tlmm 41 GPIO_ACTIVE_LOW>;
```

Comment:
ah, there was indeed a hw WDT ?

Reply:
Yes, there was, we came to know that after enquiry with them.